pkg/orchestration/wiring/asapkey/plugin.go

```diff
@@ -43,11 +45,18 @@ func New() *WiringPlugin {
 			InstanceSpec:                  instanceSpec,
 			ObjectMeta:                    objectMeta,
 			ResourceType:                  ResourceType,
-			OptionalShapes:                svccatentangler.NoOptionalShapes,
+			OptionalShapes:                optionalShapes,
```

Comment:
`Optional` in the field name probably refers to the fact that the implementor doesn't have to set this field if a resource doesn't implement any shapes, i.e. can leave it `nil`? If so, then the function implementing it should just be called `shapes`.

Not sure if my assumption is correct though.

Reply:
@scottgreenup has a PR open to change this to additional 
https://github.com/atlassian/voyager/pull/58